p/scripts/extra.js

```diff
@@ -70,13 +65,15 @@ function init_crypto_form() {
 						const s = bcrypt.hashSync(document.getElementById('passwordPlain').value, json.salt1);
 						const c = bcrypt.hashSync(json.nonce + s, strong ? bcrypt.genSaltSync(4) : poormanSalt());
 						document.getElementById('challenge').value = c;
+						submit_button.disabled = false;
 						if (!s || !c) {
 							openNotification('Crypto error!', 'bad');
 						} else {
 							crypto_form.removeEventListener('submit', crypto_form.onsubmit);
 							crypto_form.submit();
 						}
 					} catch (ex) {
+						submit_button.disabled = false;
 						openNotification('Crypto exception! ' + ex, 'bad');
 					}
 				}
```

Comment:
```suggestion
						if (!s || !c) {
							openNotification('Crypto error!', 'bad');
						} else {
							crypto_form.removeEventListener('submit', crypto_form.onsubmit);
							crypto_form.submit();
						}
					} catch (ex) {
						openNotification('Crypto exception! ' + ex, 'bad');
					}
				}
				submit_button.disabled = false;
```

A single `submit_button.disabled = false;` should be sufficient, just after the `else`

Reply:
You're right